integration-cli/docker_cli_search_test.go

```diff
@@ -11,83 +11,50 @@ func (s *DockerSuite) TestSearchOnCentralRegistry(c *check.C) {
 	testRequires(c, Network, DaemonIsLinux)
 
 	out, exitCode := dockerCmd(c, "search", "busybox")
-	if exitCode != 0 {
-		c.Fatalf("failed to search on the central registry: %s", out)
-	}
+	c.Assert(exitCode, check.Equals, 0, check.Commentf("failed to search on the central registry: %s", out))
 
-	if !strings.Contains(out, "Busybox base image.") {
-		c.Fatal("couldn't find any repository named (or containing) 'Busybox base image.'")
-	}
+	c.Assert(strings.Contains(out, "Busybox base image."), check.Equals, true, check.Commentf("couldn't find any repository named (or containing) 'Busybox base image.'"))
```

Comment:
Depending on if we integrate [shakers](https://github.com/vdemeester/shakers) or not, this package will either go, either be move in a less "integration" scoped pakage :wink:.